WordPress/src/main/java/org/wordpress/android/ui/comments/CommentAdapter.java

```diff
@@ -425,7 +425,8 @@ protected Boolean doInBackground(Void... params) {
                 return false;
             }
 
-            final int maxImageWidth = ImageUtils.getMaximumImageWidthForComment(mContext);
+            final int maxImageWidth = (int) mContext.getResources().getDimension(R.dimen.reader_comment_embedded_image_max_size); //ImageUtils.getMaximumImageWidthForComment(mContext);
```

Comment:
That works well for image based emojis, but IMO that's not a good solution for other images (especially photos)
![screenshot-2016-04-28_11 01 19 993](https://cloud.githubusercontent.com/assets/40213/14880700/9371715a-0d30-11e6-827e-184ed3e8204a.png)